dev-libs/ocl-icd/ocl_icd-2.2.12.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Alternative to vendor specific OpenCL ICD loaders"
+DESCRIPTION="OpenCL implementations are provided as ICD (Installable Client \
+Driver). An OpenCL program can use several ICD thanks to the use of an ICD \
+Loader as provided by this project. This free ICD Loader can load any (free \
+or non free) ICD."
+HOMEPAGE="http://www.blosc.org/"
+COPYRIGHT="2012-2015 Brice Videau
+	2012-2015 Vincent Danjean"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://forge.imag.fr/frs/download.php/836/ocl-icd-$portVersion.tar.gz"
+CHECKSUM_SHA256="7665f368354e3d2b7787ba4a23c6f061db1181195ba1914dd1cdcd462eca4df4"
+SOURCE_DIR="ocl-icd-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```

Comment:
Maybe:
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```
It looks like we would need a rather big patch for `ocl_icd_loader.c` to compile with gcc2, so maybe we can mark it as broken on x86_gcc2.
That said, I'm OK with keeping the `?x86_gcc2`.

Reply:
It is absolutely not needed on gcc2. But thanks for the test.